fun test(x: Int) {
    try {
        println("Početak")
        throw Exception("Greška")
    }
}
